config/commands.js

```diff
@@ -529,6 +530,42 @@ var commands = exports.commands = {
 					continue;
 				}
 			}
+
+			if (target.indexOf('>') > -1 || target.indexOf('<') > -1) {
+				if (isNotSearch) return this.sendReplyBox("Not search incompatible with stat ranges.");
+				var inequality = (target.indexOf('>') > -1 ? '>' : '<');
+				var targetParts = target.replace(/\s/g, '').split(inequality);
+				var numSide, statSide, direction;
+				if (!isNaN(targetParts[0])) {
+					numSide = 0;
+					statSide = 1;
+					direction = (inequality === '>' ? 'less' : 'greater');
+				} else if (!isNaN(targetParts[1])) {
+					numSide = 1;
+					statSide = 0;
+					direction = (inequality === '<' ? 'less' : 'greater');
+				} else {
+					return this.sendReplyBox("No value given to compare with '" + Tools.escapeHTML(target) + "'.");
```

Comment:
Having verbatim what was typed within the single quotations and nothing else makes it easiest to identify the syntax error as far as I'm concerned.